### DISCUSSION TITLE
[Kavita] padding for Mangas (Dynamic image resizing)

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2558] [Ideas]
### Idea Description

Posted by Sirius-sdz
· Apr 2022
· 12 Upvotes
https://feats.kavitareader.com/posts/99/padding-for-mangas

Description

Currently we can switch the image scaling between original, hight, width and webtoon afaik. On bigger screens I would love to have padding (for example with 10% / 20% / 30%) to find a sweet spot between way too huge (webtoon, original, width) and too tiny (height). I would love to see an implementation in the webtoon mode.

![3Hzkv9rppWQsYl0GfPCgCfCubWVkPlRKt3qfQ8S8kgtOjw7RiAhovBW2hq1gUpWj-capture](https://github.com/Kareadita/Kavita/assets/30128380/37aa6d27-1c2c-4df7-9a0c-3559f5066094)

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
A user provided a much better explanation to help me understand the need:
https://www.reddit.com/r/KavitaManga/comments/1d2qude/comment/l74ue5h/

**Comment by Marsimplodation** [comment]
hey,
got pointed here by the kavita dev
I have created a userscript with tampermonkey implementing exactly this, hope this helps
https://github.com/Marsimplodation/kavita-userscripts

https://github.com/Kareadita/Kavita/assets/46057569/35bbbeb7-62fd-4e6c-a133-b980d1bf0a26

**Reply by TEANTACLES** [reply]
I installed this, changed the domain in the header and I get the slider but it doesn't work in firefox. I can move the slider but the scaling doesn't change.

**Reply by Marsimplodation** [reply]
Have you tried the infinite scroller mode?
I am on firefox and it only works with the infinite scroll mode for me.
I 'm also not using this anymore, since I implemented it directly into kavita #2980, this still avaits review and I got told this will not make it into the next stable release, due to lack of time to test it, but should come after that sometime

But if you want you can pull this yourself, build the webapp and replace it to have the feature working on every device, with bugs fixed (like random scrolling) and a better implementation

**Reply by TEANTACLES** [reply]
Yeah, looks like it only works on the webtoon/infinite scroll reading mode, that did it, ty! I'll wait for it to get integrated into the stable build before switching off the tampermonkey extension, but ty for everything!

**Comment by majora2007** [comment]
This will be released as of v0.8.2. Thank you again @Marsimplodation for driving home the implementation.
